### DISCUSSION TITLE
<title>Regarding pushing the code for software development project I have created on my Github repository for the purpose of indicating the website link in my resume or Curriculum Vitae (CV) while applying to jobs in the field of software engineering

### DISCUSSION THREAD

**Discussion by viksof** [community/community#48549] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am applying to a job in the field of software development in a multinational tech company. I have been doing data entry or data processing kind of work until now and suddenly switching over to software engineering job function from data processing or data entry work.

I have got a Github profile at the website link https://github.com/viktech91?tab=projects where I can think about inserting or pushing code which are applicable to the software engineering projects that I will be working on.

However, I do not know exactly how my application reviewers can access the code corresponding to the software projects I will have executed from this website link.

I have to put the computer code corresponding to the software engineering projects that I will be working in the website link https://github.com/viktech91?tab=projects so that the projects that I have uploaded will be visible to the recruiters when they go through my Github profile.

May I please know how exactly can I make the  code that I have written corresponding to the software engineering projects that I have worked on in the website link https://github.com/viktech91?tab=projects so that it will be visible to the recruiters and they access it and go through it while scrutinizing my application to the software engineering jobs that I will be applying for?

Can you pls let me know about this?

**Comment by airtower-luna** [comment]
The "Projects" tab is the wrong place to look. If you want to show code you've written to the world, you need to create public repositories and push your code to them, or contribute to others' repositories. Your own repositories will show up in the "Repositories" tab, if you contributed elsewhere it might be a good idea to include links to those projects in your profile.

**Reply by viksof** [reply]
I have a public repository of mine at the website link https://github.com/viktech91?tab=repositories. May I please know how can I push the code  that I have written in this public repository of mine so that it is visible to the people visiting this repository and how can I know more about it?

May i please know about this?

**Comment by airtower-luna** [comment]
See the documentation on how to [create a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) on GitHub. If you have multiple projects you want to share, the usual approach is one repository per project. You can then upload files, or (better) use Git on your computer to commit your files and then push them to GitHub.

Your question sounds like you're not familiar with Git yet, so I strongly recommend you read the [Git Book](https://git-scm.com/book/en/v2) (it's a website, and you can download an ebook if you prefer). The first three chapters are enough to learn the basics, which you'll most likely need when working in software development. :slightly_smiling_face:

**Reply by viksof** [reply]
Hi. Thank you for the message. I got to know about a job opening called "Software developer" in a company called Rakuten in the country India from the website link https://rakuten.openings.co/#!/job-view/software-engineer-bangalore-2023020915365338 and I am seeing the following skills that are required to be possessed for working in this job role in Rakuten.

Software Engineer

Strong knowledge of web services (Restful) and design patterns
Strong microservice design concepts with Springboot/SpringCloud
Should have working experience with Redis and NoSQL
Cloud native deployment experience is an added advantage. 
Laying out the server-side design (API) with domain tables, code reviews, continuous integration & validation
Ability to write code which perform CRUD operation with Oracle database
Should be able to build feature-based implementation of APIs with good understanding of API
Proficient in JDK11 standards
Should have one or more JS framework experience (Vue, React, Angular)
Able to design modular, reusable components and apply best practices
Proficiency in GIT and other versioning tools
Experience in Agile Development environment
Proficient in unit testing using TestNG/Junit, writing mocks and automated functional tests
Shoud have working knowledge on build tools (maven) and pipeline creations using Jenkins
Good Understanding on Docker, Kubernetes, Jenkins, Sonar will be an advantage 

I am thinking to send my resume or Curriculum Vitae (CV) for this particular job opening in the company Rakuten.

Even though I have some knowledge about the skills required as indicated above, I do not know how to put them in my resume or Curriculum Vitae (CV) in order to get this job offered to me?

I thought about checking this thing with you.

May I please know whether I can indicate all the above mentioned skills in the form of a project and put it in my resume as indicated below and send my resume for making a successful application to this job opening in Rakuten India company?

Can you pls let me know about it?

[Curriculum Vitae of Vikas R (Software development engineer).odt](https://github.com/community/community/files/10857631/Curriculum.Vitae.of.Vikas.R.Software.development.engineer.odt)
